public_html/wp-content/plugins/wordcamp-speaker-feedback/includes/comment-meta.php

```diff
@@ -124,8 +133,29 @@ function get_feedback_meta_field_schema( $key = '' ) {
 				'maxlength' => 5000,
 			),
 		),
+		'helpful' => array(
+			'sft_context'       => array( 'update' ),
+			'description'       => 'The speaker found this feedback helpful.',
+			'type'              => 'boolean',
+			'single'            => true,
+			'sanitize_callback' => 'wp_validate_boolean',
+			'show_in_rest'      => false,
+			'default'           => false,
+			'required'          => false,
```

Comment:
Ah, good point. I hesitate to make it `required` because semantically it implies that all feedback should have that value, when that is not the case. However, in 4bcda31e I made it so that if there's nothing left after meta validation callback strips out invalid keys (like `q1` in your case above, which is invalid in the update context), it will throw an error instead of returning 201. Do you think that's sufficient?

Reply:
Yeah, that looks good 👍